ext/opentelemetry-ext-otcollector/README.rst

```diff
@@ -0,0 +1,55 @@
+OpenTelemetry Collector Exporter
+=============================
```

Comment:
```suggestion
OpenTelemetry Collector Exporter
================================
```

Because of

```
% python -m readme_renderer ext/opentelemetry-ext-otcollector/README.rst
<string>:2: (WARNING/2) Title underline too short.

OpenTelemetry Collector Exporter
=============================
```